src/code_translation/ir/nodes.py

```diff
@@ -100,6 +113,15 @@ def sql_string(self) -> Optional[str]:
         left_set_key = False
         if isinstance(self.left, SQLNode):
             left_sql = self.left.sql_string.strip('"').strip("'")
+        elif isinstance(self.left, JoinNode):
+            left_sql = self.left.sql_string.strip('"').strip("'")
```

Comment:
Enables multiple joins in a row. Could probably be merged into the previous if statement but I thought, this is more readable